src/main/java/com/deavensoft/paketomat/courier/CourierServiceImpl.java

```diff
@@ -66,32 +67,40 @@ public void findCitiesInRadius(String city) throws PaketomatException {
         List<City> citiesList = cityService.getAllCities();
         citiesToDispatch.clear();
 
-        for(City c : citiesList){
+        for (City c : citiesList) {
             distance = dispatcherService.findDistance(city, c.getName());
-            if(distance <= maxDistance){
+            if (distance <= maxDistance) {
                 citiesToDispatch.add(c);
             }
         }
         log.info("List with cities in 100km radius from city " + city + " is made");
     }
 
-    public List<Package> filterPackagesToDispatch(){
+    public List<Package> filterPackagesToDispatch() {
```

Comment:
The method should have List<Package> and List<City> as parameters.

Reply:
Done.